testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeCaseInsensitiveMapping.java

```diff
@@ -134,6 +134,35 @@ public void testColumnCommentWithNonLowerCaseColumnName()
         }
     }
 
+    @Test(groups = {DELTA_LAKE_OSS, PROFILE_SPECIFIC_TESTS})
+    public void testViewColumnCommentWithNonLowerCaseColumnNameAlias()
```

Comment:
We should avoid product tests as much as possible. https://trino.io/docs/current/develop/tests.html#avoid-product-tests

Is it possible to test in engine (e.g. TestCommentTask)?

Reply:
Big thx! Refactored